agent/structs/connect_proxy_config.go

```diff
@@ -242,16 +250,18 @@ func (t *ConnectProxyConfig) UnmarshalJSON(data []byte) (err error) {
 }
 
 func (c *ConnectProxyConfig) MarshalJSON() ([]byte, error) {
-	type typeCopy ConnectProxyConfig
-	copy := typeCopy(*c)
-
-	proxyConfig, err := lib.MapWalk(copy.Config)
```

Comment:
Did you still need this `lib.MapWalk` call?

Reply:
Yep, forgot to add it back in. Just brought it back.